### ISSUE TITLE
SQLite/tRPC works in dev but not in prod enviroment

### ISSUE THREAD

**Issue by evvarela** [AyanavaKarmakar/tauri-t3-app#19]
Hello!

First of all thanks for this amazing work!

In dev enviroment I cant make queries and mutate using a SQLite database, but when I compile the app my tRPC routes doesn't work.
![image](https://user-images.githubusercontent.com/73674669/236382280-fddad698-ec40-4acc-aa92-a9c1cd3e8cef.png)

**Comment by c2rlost2r** [comment]
Yes, the same thing happens with PostgreSQL. I suppose it's because Tauri uses frontend libraries like React and in backend uses Rust. To use Prisma in Tauri, there is the prisma-client-rust library.

**Comment by rom3N123** [comment]
has someone found a fix?
